subprojects/test-runner/report-viewer/src/test/kotlin/com/avito/report/model/RunIdTest.kt

```diff
@@ -1,13 +1,13 @@
 package com.avito.report.model
 
 import com.google.common.truth.Truth.assertThat
-import org.junit.Test
+import org.junit.jupiter.api.Test
```

Comment:
tests didn't work

Reply:
Checked all other places in project, all is ok.
Static analysis check will be a nice addition, but i don't know how to do it rn.

Reply:
MBS-11359